Dockerfile

```diff
@@ -1,35 +1,23 @@
 FROM node:10-alpine as deps
-
 COPY package.json yarn.lock ./
-
-RUN yarn && \
-  yarn install --modules-folder node_modules_prod
+RUN yarn
+RUN yarn install --modules-folder node_modules_prod
```

Comment:
I think it used to be best practice to actually group multiple RUN commands because it reduces the number of layers, not sure how relevant that still is. Also shouldn't `NODE_ENV` be set to `"production"` for the prod modules?

Reply:
done